packages/components/psammead-brand/README.md

```diff
@@ -0,0 +1,15 @@
+# Brand [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Brand)	
```

Comment:
This ought to have the badges that other components have: https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-visually-hidden-text

(I appreciate this was copied & pasted)